pages/models.py

```diff
@@ -322,6 +322,12 @@ class CandidateRace(models.Model):
             ('lose', 'Lose'),
         )
     candidate = models.ForeignKey('endorsements.Candidate', null=True, blank=True, on_delete=models.SET_NULL)
+    candidate_endorsement_page = models.ForeignKey(
```

Comment:
So problem here is that we'll have to manually set pages for over 100 races, as they do not auto-populate. What I recommend instead is accessing the candidate slug on the template side like `<a href="/candidates/{{ candidate_race.candidate.slug }}">` and avoiding any model changes or migrations.

Reply:
Unfortunately slug is not reliable for all cases. I suggest handling the backfill of old results in a future issue. The most important races are the new ones, and we can set them as we add them going forward. I agree slug is easy and automatic but some of them will be wrong urls because candidate slug is separate field in db from page url.

Reply:
Agreed about slugs being unreliable but problem with back filling is that we're going to have only partial results be links which is going to be frustrating to user and they're going to think it's a bug. Slug is generated same way it is generated through wagtail so the likelihood of them being different is small, and we can run quick python script to verify they are all the same. We also add step to the add results process which isn't intuitive.

Is there a solution where we don't have to have only some cards on page be links? Perhaps a custom.migration, or perhaps we just verify slug and then make future ticket for scaleable solution?

Reply:
I suggest we do one of 2 options:
1. make a ticket to update old results to select pages for each of them
2. make slug url the default link if there is no page selected, and page select can just be backup option if slug fails

I suggest 1 because in long term we can replace candidate field with candidate page field, and also I dont understand why candidate model has slug field to begin with since it is not used?

Reply:
I'm fine with adding a conditional to fallback to slug if there's no candidate page, and then we can make a ticket for updating old results.

We can refactor this process post-election.